src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -289,6 +289,82 @@ begin
   rw (congr_fun (congr_arg (@nhds E) hp.1) 0),
   exact add_group_filter_basis.nhds_zero_has_basis _,
 end
+
+/-- The `x`-neighbourhoods of a space whose topology is induced by a family of seminorms
+are exactly the sets which contain seminorm balls around `x`.-/
+lemma with_seminorms.mem_nhds_iff (hp : with_seminorms p) (x : E) (U : set E):
+  U ∈ nhds x ↔ ∃ (i : finset ι) (ε > 0), (i.sup p).ball x ε ⊆ U :=
```

Comment:
```suggestion
  U ∈ nhds x ↔ ∃ (s : finset ι) (ε > 0), (s.sup p).ball x ε ⊆ U :=
```
I am pretty sure we used `s` as the primary variable for the finset almost everywhere.